[2021-09-01 00:42:50,034] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-09-01T00:42:28.717277+00:00 [queued]>
[2021-09-01 00:42:50,095] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-09-01T00:42:28.717277+00:00 [queued]>
[2021-09-01 00:42:50,098] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-09-01 00:42:50,099] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-09-01 00:42:50,100] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-09-01 00:42:50,135] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_4> on 2021-09-01T00:42:28.717277+00:00
[2021-09-01 00:42:50,148] {standard_task_runner.py:52} INFO - Started process 97584 to run task
[2021-09-01 00:42:50,178] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', '2021-09-01T00:42:28.717277+00:00', '--job-id', '14', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmppnwaymwy', '--error-file', '/tmp/tmpt4gazrqf']
[2021-09-01 00:42:50,180] {standard_task_runner.py:77} INFO - Job 14: Subtask task_4
[2021-09-01 00:42:50,482] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_4 2021-09-01T00:42:28.717277+00:00 [running]> on host airflowvm
[2021-09-01 00:42:50,875] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2021-09-01T00:42:28.717277+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-09-01T00:42:28.717277+00:00
[2021-09-01 00:42:50,877] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-09-01 00:42:50,879] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-09-01 00:42:50,893] {subprocess.py:75} INFO - Output:
[2021-09-01 00:42:53,914] {subprocess.py:83} INFO - Command exited with return code 0
[2021-09-01 00:42:53,988] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20210901T004228, start_date=20210901T004250, end_date=20210901T004253
[2021-09-01 00:42:54,067] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-09-01 00:42:54,113] {local_task_job.py:151} INFO - Task exited with return code 0
